src/about-menu/open-about-menu.ts

```diff
@@ -39,7 +40,14 @@ export function openAboutWindow() {
 		if ( aboutWindow ) {
 			// Inject version into the about window's HTML
 			aboutWindow.webContents
-				.executeJavaScript( `document.getElementById('version').innerText = '${ packageJson }'` )
+				.executeJavaScript(
+					`document.getElementById('version').innerText = '${ packageJson }'; 
+					document.getElementById('studio-by-wpcom').innerText = '${ __( 'Studio by WordPress.com' ) }';
+					document.getElementById('version-text').innerText = '${ __( 'Version' ) }'; 
+					document.getElementById('share-feedback').innerText = '${ __( 'Share Feedback' ) }';
+					document.getElementById('demo-sites').innerText = '${ __( 'Demo sites powered by' ) }';
+					document.getElementById('local-sites').innerText = '${ __( 'Local sites powered by' ) }';`
+				)
 				.catch( ( err ) => {
 					console.error( 'Error executing JavaScript:', err );
```

Comment:
Should we capture the error in sentry?

Reply:
Added in https://github.com/Automattic/studio/pull/228/commits/343ac4d9fd5925e303113b5d28843bf74d4f4b7b